src/functions.js

```diff
@@ -1,19 +1,17 @@
-const scores = [];
-
-const addScore = (name, score) => {
-  scores.push({ name, score });
-  return { name, score };
-};
-
-const createScore = ({ name, score }) => {
-  const h4 = document.createElement('h4');
-  h4.classList.add('custom-row', 'p-1');
-  h4.textContent = `${name}: ${score}`;
-  document.querySelector('.display').appendChild(h4);
-};
-
-const getDisplay = () => {
-  scores.forEach(createScore);
+const createGame = async () => {
+  await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/101/scores', {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: document.querySelector('.name').value,
+      score: document.querySelector('.score').value,
+    }),
+  })
+    .then((response) => response.json())
+    .then((data) => console.log(data)) // eslint-disable-line no-console
```

Comment:
- [ ] Please erase all the `console.log` from your application, this will make your code look cleaner 😄